migrations/env.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import with_statement
```

Comment:
What does this have to do with logging?

Reply:
It's not related to logging per se, but it's something that hasn't been needed since python 2.6.